requirements/requirements.txt

```diff
@@ -1,6 +1,6 @@
-pandas<0.23
-requests==2.24.0
-urllib3==1.24.3
-websocket-client==0.57.0
-websockets>=8.0
-alpha_vantage==2.2.0
+pandas>=0.18.1
```

Comment:
I would agree more to loosen other packages. Do we want to support both pandas 0.x and 1.x?

Reply:
we could with:
pandas 
or
pandas>1

and let pylivetrader downgrade it

Reply:
I'm inclined to do >1 then.

Reply:
But does it actually work? I think pip will reject such install?

Reply:
If pylivetrader can't support pandas>1, then maybe we need to do >=0.xx

Reply:
pip will give a warning but will install correctly (for pylivetrader it will remove 1.0.5 and install 0.22.0)